<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCard</class>
 <widget class="QWidget" name="NewCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>790</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>790</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>790</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="minimumSize">
        <size>
         <width>842</width>
         <height>458</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>900</width>
         <height>800</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#stackedWidget{border-image: url(:/Image/Bkgrnd.png);}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_Waitcursor">
           <property name="minimumSize">
            <size>
             <width>63</width>
             <height>61</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>61</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/Image/Countdown.png);
font: 57 42pt &quot;思源黑体 CN Medium&quot;;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_CountDown">
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: normal;
line-height: 49px;
letter-spacing: 1px;
color: #707070;
font:42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_MainPage">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>61</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>61</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/Image/Homepage.png);
image-position: left;
padding-left: 2px;
text-align: right;
padding-right: 2px;
font-size: 12px;
font: 28pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_step">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>348</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>348</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Image/updatecard1.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>140</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
